### PR TITLE
ui: Clarify menu item for moving

### PR DESCRIPTION
**Pull request by nijel**
## Proposed changes

This hides both renaming and moving to different project or category, so the menu item should be clearer here.

See #9807

## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

### CODE REVIEW

**Comment by nijel** [comment]
This is where it points to:

image

**Comment by orangesunny** [comment]
Let's join it with Remove menu and call the menu item 
**Organize or remove**

**Comment by nijel** [comment]
Good idea, I've changed the PR to implement that.

**Review by m-aciek — APPROVED** [review]

File: weblate/templates/trans/delete-form.html [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {% load i18n %}
 {% load crispy_forms_tags %}
-<div class="tab-pane" id="delete">
```
Comment (m-aciek) [inline weblate/templates/trans/delete-form.html:3]: nit: this newline seems superfluous now

**Pull request merged**